index.js

```diff
@@ -1,9 +1,7 @@
-import {RadCommonGlobalSearch} from "./src/radCommonGlobalSearch";
-import {RadCommonAutoComplete} from "./src/radCommonAutoComplete";
-
-import $ from 'jquery'
```

Comment:
I'm assuming we are importing this elsewhere? Will look for it as I get farther down the code